src/components/Superlatives.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Nav from './Nav.jsx';
+import Student from './Superlative.jsx'
+
+export default class Superlatives extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div>
+        <Nav />
+
+        <div className="students list container column">
```

Comment:
This will display superlatives instead of students. It can wait till a future PR, when we add the GET request

Reply:
Oh yeah... I kinda copy pasted for the format, but then I didn't make it superlatives because I didn't want to have too many commits in a single pull request... but I should fix this in this pull request.